feature: back to top button

kKaskak opened the pull request:
Added a back to top button in `StreamsList` component. Converted to draft for now.

kKaskak committed:
fix: streams-list

kKaskak committed:
update: spacing and icon added

kKaskak committed:
feature: scrollTop

kKaskak committed:
refactor: scroll top button

kKaskak committed:
refactor: changed positioning of back-to-top button

kKaskak committed:
refactor: changed the postioning of the back to top button & animation added to addons-container

kKaskak committed:
refactor: useLayoutEffect instead of useEffect

kKaskak committed:
refactor: add delay to the loading addons popup

kKaskak committed:
delete: unneccessary empty line in StreamsList.js

kKaskak committed:
refactor: simplify the logic of StreamsList div

kKaskak committed:
refactor: change naming for the usestate

unclekingpin commented:
What issue does this pr solve? can you provide a screenshot of the UI before and after?

kKaskak commented:
> What issue does this pr solve? can you provide a screenshot of the UI before and after?

These are mostly iPad related fixes in font-size etc. Fixed the jump created by the addons loading container by moving it to the bottom of the StreamsList. Added a back to top button if the container is longer. Moved the postion of the fixed install ddons button to conditional rendering based on the length of the streamsList. 
Before --> 
![IMG_3843](https://github.com/Stremio/stremio-web/assets/117831817/209b4bb1-84e3-4f82-bd47-1443624d0e44)
After -->
![IMG_3842](https://github.com/Stremio/stremio-web/assets/117831817/7eb4d681-6ee5-496b-99e1-6eda62f42bfa)
![IMG_3841](https://github.com/Stremio/stremio-web/assets/117831817/7bef0997-e13b-4e5a-aa76-3baafd269431)
Reference video: 
https://github.com/Stremio/stremio-web/assets/117831817/0a24de14-a6b6-4e05-ba6f-056efb0ecdd3

kKaskak committed:
Merge branch 'development' into fix-details-page-streams-list

kKaskak committed:
Update StreamsList.js
